docs/_posts/examples/3400-01-02-heatmap.html

```diff
@@ -21,15 +21,15 @@
 
     // Add a new source from our GeoJSON data and set the
     // 'cluster' option to true.
-    map.addSource("earthquakes", {
+    var earthquakes = {
```

Comment:
I don't think we should change examples such as this, where multiple layers use a single source.